collection-scripts/gather_kube_apiserver_termination_logs

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+TERMINATION_LOG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/kube_apiserver_termination_logs"}
+
+mkdir -p "${TERMINATION_LOG_PATH}"/
+
+function get_dump_off_node {
+    local debugPod=""
+    
+    #Get debug pod's name
+    debugPod=$(oc debug --to-namespace="default" node/"$1" -o jsonpath='{.metadata.name}')
+
+    #Start Debug pod force it to stay up until removed in "default" namespace
+    oc debug --to-namespace="default" node/"$1" -- /bin/bash -c 'sleep 300' > /dev/null 2>&1 &
+
+    #Mimic a normal oc call, i.e pause between two successive calls to allow pod to register
+    sleep 2
+    oc wait -n "default" --for=condition=Ready pod/"$debugPod" --timeout=30s
+
+    if [ -z "$debugPod" ]
+    then
+      echo "Debug pod for node ""$1"" never activated"
+    else
+      #Copy API watch log
+      echo "Copying termination log on node ""$1"""
+      oc cp  --loglevel 1 -n "default" "$debugPod":/host/var/log/kube-apiserver/termination.log "${TERMINATION_LOG_PATH}"/"$1"_termination.log > /dev/null 2>&1 && PIDS+=($!)
+
+      #clean up debug pod after we are done using them  
+      oc delete pod "$debugPod" -n "default"  
+    fi
+}
+
+function gather_api_watchlog_data {
+  #Run watchlog pull function on all nodes in parallel
+  for NODE in ${NODES}; do
+    get_dump_off_node "${NODE}" &
+  done
+}
+
+if [ $# -eq 0 ]; then
+    echo "WARNING: Collecting watchlog dumps on ALL linux nodes in your cluster. This could take a long time."
```

Comment:
update all this text too, I have no idea what a watchlog is (and this is all output someone will have to use to find this script)

Reply:
Updated to

```
WARNING: Collecting kube apiserver watch termination logs on ALL linux nodes in your cluster.
```